tasks/sc2.py

```diff
@@ -224,7 +224,7 @@ def deploy(ctx, debug=False, clean=False):
     # Fail-fast if we are not using the expected host kernel
     host_kernel_version = get_host_kernel_version()
     host_kernel_expected_prefix = get_host_kernel_expected_prefix()
-    if not host_kernel_version.startswith(host_kernel_expected_prefix):
+    if not host_kernel_version.startswith("6.8.0") and host_kernel_version.endswith("intel"):
```

Comment:
This would only work for tdx, but not for SNP. I would suggest adding another method: `get_host_kernel_expected_suffix()` in `tasks/util/kernel.py` that returns nothing for SNP, and `intel` for TDX. You must also change the `prefix` function to return the right prefix depending on the HW.